### PR TITLE
Fixed issue #16799: User with read + create survey : invalid list shown

### PR DESCRIPTION
**Pull request by gabrieljenik**
LIsting survey which have at least read permissions

### CODE REVIEW

File: application/controllers/admin/surveyadmin.php [hunk 1]
```diff
@@ -1265,6 +1265,10 @@ public function editlocalsettings($iSurveyID)
      */
     public function copy()
     {
+        if (!Permission::model()->hasGlobalPermission('surveys', 'create')) {
```
Comment (Shnoulle) [inline application/controllers/admin/surveyadmin.php:1268]: 😢 403 error …

Comment (gabrieljenik) [reply application/controllers/admin/surveyadmin.php:1268]: What do you mean?

Comment (Shnoulle) [reply application/controllers/admin/surveyadmin.php:1268]: Send a 403 error : https://bugs.limesurvey.org/view.php?id=16470

Some reason why it's better are in the comments , some issue take really more time without a clean error : https://bugs.limesurvey.org/view.php?id=16470#c60490

Comment (gabrieljenik) [reply application/controllers/admin/surveyadmin.php:1268]: Done

**Comment by Shnoulle** [comment]
Thanks you !!! Realy thank you ! (because it's not mandatory ;)

**Review by Shnoulle — APPROVED** [review]
:+1: :heart:

**Pull request merged**